lib/dotiw/action_view/helpers/date_helper.rb

```diff
@@ -15,6 +15,7 @@ def distance_of_time_in_words(from_time, to_time = 0, include_seconds_or_options
 
       def distance_of_time_in_percent(from_time, current_time, to_time, options = {})
         options[:precision] ||= 0
+        options = options_with_scope(options)
```

Comment:
We should fix the fact that we're modifying incoming `options` hash here and in other places. For a separate PR if you feel motivated, https://github.com/radar/distance_of_time_in_words/issues/113.